tests/system-tests/rdscore/README.md

```
@@ -313,3 +581,270 @@ and firewall traffic or probes sufficient to produce journal lines**
 ### _VerifyNMStateInstanceExists_
 
 Verifies that _NMState_ instance `nmstate` exists
+
+## Environment Variables
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)